providers/base/bin/virtualization.py

```diff
@@ -842,8 +843,8 @@ def cleanup(self):
         Clean up test files an Virtual Machines created
         """
         logging.debug('Cleaning up images and VMs created during test')
-        self.run_command('lxc image delete {}'.format(self.image_alias))
-        self.run_command('lxc delete --force {}'.format(self.name))
+        self.run_command('lxc image delete {}'.format(self.image_alias), False)
+        self.run_command('lxc delete --force {}'.format(self.name), False)
```

Comment:
Minor and possibly out of the scope of this PR: it is a very strange behavior to use strings everywhere and then split them with `shlex.split`, this should be changed in the future to use arrays everywhere IMO